packages/nodes-base/package.json

```diff
@@ -707,14 +711,19 @@
     "@types/nodemailer": "^6.4.0",
     "@types/redis": "^2.8.11",
     "@types/request-promise-native": "~1.0.15",
+    "@types/shelljs": "0.8.11",
     "@types/ssh2-sftp-client": "^5.1.0",
     "@types/tmp": "^0.2.0",
     "@types/uuid": "^8.3.2",
     "@types/xml2js": "^0.4.3",
+    "esbuild": "0.14.34",
     "gulp": "^4.0.0",
     "jest": "^27.4.7",
     "n8n-workflow": "~0.94.0",
     "nodelinter": "^0.1.9",
+    "npm-run-all": "4.1.5",
+    "shelljs": "0.8.5",
+    "tiny-glob": "0.2.9",
```

Comment:
`tiny-glob` in /nodes-base vs. `fast-glob` in /cli. We should use a single lib - maybe tech debt to switch /cli to `tiny-glob` if that's the one we want to keep.

Reply:
I chose `tiny-glob` because it's much faster (~230%) than `fast-glob`. I don't have a strong preference, but the syntax is the same for both. Should be straightforward to switch in `/cli` as well.